Metro-Records/la-metro-councilmatic
# Move search bar and popover

## Overview

After getting approval on [this comment](https://github.com/datamade/la-metro-councilmatic/issues/482#issuecomment-1416169503), the search bar and _Learn More_ tooltip have been moved to improve UX and make the search bar more of a highlighted feature.

Closes #943 

### Demo

![image](https://user-images.githubusercontent.com/114717958/220144484-5e46d3b4-05cd-44f6-9b35-497c60097751.png)
![image](https://user-images.githubusercontent.com/114717958/220144542-1a31185a-1d8e-45e5-8405-95b4655c2f3b.png)


### Notes

I expected the popover to follow where the _Learn More_ link was, but it stayed in place at the top of the page. So I added a top margin to the css of popovers since, doing a search through the files showed that here was the only place a popover was used like this. Because of that I'm not expecting for that margin to mess with anything else, but if it does then I can absolutely figure something else out for the positioning!

## Testing Instructions

 * Navigate to board of directors page
 * Check the styling/positioning of the search bar and _Learn More_ tooltip
